src/data/completedTasks.tsx

```diff
@@ -0,0 +1,123 @@
+export type Tasks = {
```

Comment:
From the catch up with Dan yesterday the information we are storing on the CompletedTask has changed slightly. 

Can you make your mock data follow this schema/structure? 

You can find it on the [utility-mock-data](https://github.com/nology-tech/wov-client-project/blob/utility-mock-data/src/mockData/mockCompletedTasks.ts) branch.

Reply:
We've added the right mock data